system/Test/FeatureTestTrait.php

```diff
@@ -51,10 +51,16 @@ protected function withRoutes(?array $routes = null)
             $collection->resetRoutes();
 
             foreach ($routes as $route) {
+                /**
+                 * @TODO For backward compatibility. Remove strtolower() in the future.
+                 * @deprecated 4.5.0
+                 */
+                $method = strtolower($route[0]);
```

Comment:
If you go with @paulbalandan's trigger idea this would be another good place for a conditional warning.

Reply:
Done.